modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -3311,9 +3311,8 @@ CV_EXPORTS_W void filterHomographyDecompByVisibleRefpoints(InputArrayOfArrays ro
 class CV_EXPORTS_W StereoMatcher : public Algorithm
 {
 public:
-    enum { DISP_SHIFT = 4,
-           DISP_SCALE = (1 << DISP_SHIFT)
-         };
+    static constexpr auto DISP_SHIFT = 4;
+    static constexpr auto DISP_SCALE = 1 << DISP_SHIFT;
```

Comment:
I propose not touch enums for today. It breaks current bindings generator. It may introduce issues with old compilers.

Reply:
Thank you for your review! I added some fixes in arithmetic statement with some enums and float numbers to suppress warnings.